### PR TITLE
medical/clown rigs + weaponry/ammunition, store/SO revamp

### PR DESCRIPTION
**Pull request by karlodayo**
I've added and visually updated medical rig, and implemented the clown rig. The store kiosk and supply management is getting a revamp on the weaponry/security side of things. This will overall allow for more roleplay for Supply Officer and medical staff.

### CODE REVIEW

**Comment by nianoru** [comment]
The commits above are simply visual/sprite changes. In-game implementation will be put in Viz's PR, as he decided to go forth and do it to prevent conflict.

**Comment by nianoru** [comment]
ready for review

**Comment by nianoru** [comment]
for the medical rig commit, it's simply visual/sprite changes. In-game implementation will be put in Viz's PR, as he decided to go forth and do it to prevent conflict. This PR is ready for review.

File: code/modules/clothing/spacesuits/rig/suits/civilian.dm [hunk 1]
```diff
@@ -47,3 +47,33 @@
 
 /obj/item/clothing/head/helmet/space/rig/emergency
 	name = "helmet"
+
+/obj/item/rig/clown
+	name = "clown RIG"
+	desc = "honk honk."
```
Comment (DTraitor) [inline code/modules/clothing/spacesuits/rig/suits/civilian.dm]: Description should start with a capital letter

**Review by DTraitor — CHANGES_REQUESTED** [review]
Just some small tweaks. Remove `medical_rig_old` sprites, if we ever need old sprites we will use github commit history. HUD sprite of the RIG in `icons/obj/rig_frames.dmi` should be `Back_in_inv.png`, not a bike horn sprite.

**Review by DTraitor — APPROVED** [review]

**Comment by nianoru** [comment]
Not sure if Viz left a changelog of his own in code. If you need me to add it into this PR before closing it, let me know via discord or here.

**Comment by nianoru** [comment]
Adding store kiosk/supply management revamp before this friday! :)

**Comment by nianoru** [comment]
PR finished.

**Comment by nianoru** [comment]
the last 3 shoulda been one commit, had a troll moment. anyways.. PR is finished woo.

**Comment by karlodayo** [comment]
the last 3 shoulda been one commit, had a troll moment and its also 5am. anyways.. PR is finished woo.. :')

**Review by DTraitor — APPROVED** [review]

**Pull request merged**